recipes/sekve/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sekve" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f6891935794e356ef890fe30547dfcc3a1197ec01a4f588de615e039a6d51b29
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - matplotlib-base =3.5.1
+    - numpy =1.22.3
+    - pandas =1.4.2
+    - scipy =1.7.3
+
```

Comment:
These are super strict and prone to cause resolve conflicts in the future. Maybe it is worth opening an issue upstream.